tx-indexer/src/config.rs

```diff
@@ -503,10 +515,7 @@ impl ScyllaDBManager {
         let mut result = std::collections::HashMap::new();
         let mut rows_stream = self
             .scylla_session
-            .query_iter(
-                "SELECT transaction_details FROM tx_indexer_cache.transactions",
-                &[],
-            )
+            .execute_iter(self.cache_get_transactions.clone(), &[])
```

Comment:
I think it's better to make these functions on the ScyllaManager side like we did for others to call it ScyllaManager::cache_get_transactions() instead of this mess. Wdyt?